src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagService.cs

```diff
@@ -33,7 +33,7 @@ public CustomTagService(ICustomTagStore customTagStore, IReindexJob reindexJob,
             _logger = logger;
         }
 
-        public async Task<AddCustomTagResponse> AddCustomTagAsync(IEnumerable<CustomTagEntry> customTags, CancellationToken cancellationToken = default)
```

Comment:
> = default) [](start = 141, length = 11)

Any reason for removing the default?

Reply:
I think we require the cancellation token to be provided in implementations! Based on other similar classes.

---
In reply to: [564818697](https://github.com/microsoft/dicom-server/pull/501#discussion_r564818697) [](ancestors = 564818697)

Reply:
Oh ok

---
In reply to: [566993274](https://github.com/microsoft/dicom-server/pull/501#discussion_r566993274) [](ancestors = 566993274,564818697)